packages/host/tests/acceptance/operator-mode-test.ts

```diff
@@ -1132,7 +1143,15 @@ module('Acceptance | operator mode tests', function (hooks) {
       assert.strictEqual(json.data.attributes?.name, 'MangoXXX');
     });
 
-    setMonacoContent(JSON.stringify(expected));
+    await this.expectEvents(
+      assert,
+      realm,
+      adapter,
+      expectedEvents,
+      async () => {
+        setMonacoContent(JSON.stringify(expected));
```

Comment:
This diff reads a bit weirdly to me, there‘s an expectation but the callback doesn’t seem to relate to it? Could the callback be optional?

Reply:
The diff is that I added the `expectEvents()` which internally has an test assertion (which is why it was increased)

Reply:
👍🏻 understood, thanks